packages/remark-images-download/src/index.js

```diff
@@ -1,3 +1,6 @@
+require('core-js/stable')
+require('regenerator-runtime/runtime')
```

Comment:
Both of these seem overkill. Upgrading this package independently as you suggested makes sense IMO, I'd suggest reverting this file and only bumping unist-util-visit/rimraf/shortid for now.